tools/cli/NearObjectCli.cxx

```diff
@@ -254,14 +260,17 @@ NearObjectCli::AddSubcommandUwbRangeStart(CLI::App *parent)
         }
 
         if (!m_cliData->ResultReportConfigurationString.empty()) {
-            auto result = uwb::protocol::fira::StringToResultReportConfiguration(m_cliData->ResultReportConfigurationString, detail::ResultReportConfigurationMap);
+            auto result = uwb::protocol::fira::StringToResultReportConfiguration(m_cliData->ResultReportConfigurationString, ::detail::ResultReportConfigurationMap);
             if (not result) {
                 std::cout << "could not parse ResultReportConfiguration" << std::endl;
             } else {
                 m_cliData->SessionData.uwbConfiguration._resultReportConfigurations = result.value();
             }
         }
 
+        m_cliData->SessionData.staticRangingInfo = m_cliData->StaticRanging;
```

Comment:
Is there any reason not to use `m_cliData->SessionData.staticRangingInfo` as the `assignTo` variable directly instead of having this extra level of indirection and copying?

Reply:
I wasn't sure if you could write to a std::optional that is a std::nullopt but I will try it

Reply:
Just verified that you can't directly write to a member of a class wrapped in a std::optional if it is not allocated. But maybe I can initialize the std::optional as a default initialized StaticRangingInfo and then write to those members?

Reply:
Ahh yes, I see that the member itself is a `std::optional` and CLI11 doesn't support any indirection.